hdfscontents/hdfsmanager.py

```diff
@@ -17,6 +17,8 @@
 import mimetypes
 import nbformat
 from traitlets import Instance, Integer, Unicode, default
+from hops import xattr
```

Comment:
As we want hdfscontents to work in environments where hops is not installed, can you add a try here, and log the exception?